pkgs/by-name/po/polylux2pdfpc/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, nix-update-script }:
+
+let
+  dirname = "pdfpc-extractor";
+in
+rustPlatform.buildRustPackage rec {
+  pname = "polylux2pdfpc";
+  version = "0.3.1";
+
+  src = (fetchFromGitHub {
+    owner = "andreasKroepelin";
+    repo = "polylux";
+    rev = "v${version}";
+    sparseCheckout = [ dirname ];
+    hash = "sha256-GefX7XsUfOMCp2THstSizRGpKAoq7yquVukWQjGuFgc=";
+  }) + "/${dirname}";
```

Comment:
Aren't we supposed to use `sourceRoot` here ?

Reply:
I knew there had to be a better way, TIL :)
Thank you!

I've left the sparse checkout, do you think I should take it out as well?

Reply:
I never used it yet, so I don't know